components/Topbar.tsx

```diff
@@ -11,7 +11,7 @@ export default function Topbar({ items }: { items: NavItem[] }) {
     <div className="absolute left-0 top-0 w-full">
       <div className="m-auto flex items-center justify-between px-6 py-4 text-white">
         <Link
-          href="/"
+          href="/en"
```

Comment:
This should be dynamic though, if a user is using another language we don't want to force the `en` version, so this is not the way we should fix this. In the `middleware.ts` file we redirect the user to their preferred locale, but something is wrong if the target page is the index file, needs some further investigation there.

Reply:
Yes, I saw the changes you did here - https://github.com/saving-satoshi/saving-satoshi/pull/151/commits/239f7715ee29064a26637b1c73d3826b899d02cf in order to solve this issue and understood that it requires much more investigation than what I did before creating this PR. One solution that came to my mind is to redirect the user using the selected `locale` or we can create an expectation when we are redirecting to the homepage as this issue is only occurring when redirecting to the homepage (This only occurs in production server and not in local development enviroment)

Reply:
Yeah, I also tried reproducing this locally and wasn't able to. I think we can use your first suggestion, so we pass the `lang` as a prop to the `<Topbar lang={lang} />` component and redirect users to `/${lang}` when they click the title. I think things will have to change anyway once the Next 13 app directory approach matures a little bit more, so this is fine as a temporary fix.